appendix.Rmd

```diff
@@ -2,8 +2,52 @@
 
 # NEWS {#booknews}
 
+## dev
+
+* 2019-09-26, add instructions for handling editors to nominate packages for blog posts (ropensci/dev_guide#180)
+
+* 2019-09-26, add chapter on changing package maintainers (ropensci/dev_guide#128) (ropensci/dev_guide#194)
```

Comment:
Why put ropensci/dev_guide instead of just the PR number? Genuinely curious.

Reply:
we have some issues that are from software-review-meta and some from dev_guide. so unless there's a org/repo before the issue number, there's no way to tell which repo the issue is from